cryostat/Chart.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: v2
+name: cryostat
+description: Securely manage JFR recordings for your containerized Java workloads
+
+# A chart can be either an 'application' or a 'library' chart.
```

Comment:
guess you'd like to remove the default comments.

Reply:
Definitely, thanks for the reminder!